hatchery/src/world.rs

```diff
@@ -193,6 +200,17 @@ impl World {
         Ok(id)
     }
 
+    /// Deploys a [`HostQuery`] with the given module ID.
```

Comment:
given module id?

Reply:
When I first started I wanted host queries to piggy back on the normal query infra. Forgot to update the comment. Well spotted.